tests/wdio.conf.ts

```diff
@@ -1,3 +1,5 @@
+/* global process, browser, Buffer */
```

Comment:
since this runs with node you can import process and Buffer. Perhaps even browser too, but that's from wdio.

Reply:
will see